Makefile

```diff
@@ -0,0 +1,38 @@
+SEVERITIES = HIGH,CRITICAL
+
+ifeq ($(ARCH),)
+ARCH=$(shell go env GOARCH)
+endif
+
+ORG ?= rancher
+TAG ?= v0
+REPO=opni-metric-anomaly-detector
```

Comment:
Can we just call this opni-metrics-service? This naming would match the other services

Reply:
updated